pkg/app/appdisc/discovery_updater.go

```diff
@@ -29,8 +29,8 @@ func (emptyUpdater) Start()                                  {}
 func (emptyUpdater) Stop()                                   {}
 func (emptyUpdater) ChangeValue(name string, v []byte) error { return nil }
 
-// proxyUpdater updates proxy-discovery entry of locally running skysocks App.
-type proxyUpdater struct {
+// serviceUpdater updates service-discovery entry of locally running App.
```

Comment:
Service discovery is still named proxy discovery throughout the project. I think we need to change the occurrences either here or in this PR: https://github.com/SkycoinProject/skywire-mainnet/pull/393. Please let me know your decision.

Reply:
This is just one thing I bumped into, not sure that here's the right place to fix it. so, go ahead with it